ios/BetterSocial.xcodeproj/project.pbxproj

```diff
@@ -726,8 +721,8 @@
 			buildPhases = (
 				D3DED40E04AAC09834B084E6 /* [CP] Check Pods Manifest.lock */,
 				C4AE08FE2AC43A4800E13758 /* Sources */,
-				C4AE08FF2AC43A4800E13758 /* Frameworks */,
 				C4AE09002AC43A4800E13758 /* Resources */,
+				6A774A4C9B67562633967EEE /* Frameworks */,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The order of the build phases has changed here. As long as this doesn't affect the build process, this should be fine.



<!-- This is an auto-generated comment by OSS CodeRabbit -->